openfisca-web-api/input-output-data.md

```diff
@@ -1,104 +1,294 @@
-# Input and output data
+# Using the /calculate endpoint
 
-## Values by period
+> All the examples provided here are from the [country package template](https://github.com/openfisca/country-template).
 
-In OpenFisca, the value of a variable is always defined for a specific period.
+In order to run a computation on the Web API, you will need to send information to the API concerning:
+- The situation, meaning describe the entities (e.g. individuals, households) that you want to base your calculations on.
+- The variable you need to compute.
 
-This is encoded in JSON using the following format:
+## Describing the situation
 
-```JSON
-{"salary": {"2015-01": 2000}}
-```
+### Describing entities
+
+The most important rule in describing a situation in OpenFisca is:
+
+Every person has to belong to one of each group entity (e.g. household).
+Every person in a group entity needs a role (e.g. parent)
 
-This format allows you to encode an evolution of the variable over time:
+> For example, if you wish to run a calculation on 2 households. One composed of to adults, the other of one adult and one child.
 
-```JSON
+
+```json
 {
-  "salary": {
-    "2015-01": 2000,
-    "2015-02": 2000,
-    "2015-03": 2100,
-    "2015-04": 2100,
+  "persons": { 
+    "Ricarda": {},
+    "Bob": {},
+    "Bill": {},
+    "Janet": {}
+    },
+  "households": {
+    "household_1": {
+      "parents": [
+        "Ricarda", "Bob"
+      ]
+    },
+    "household_2": {
+      "parents": [
+        "Bill"
+      ],
+      "children":[
+        "Janet"
+      ]
+    }
   }
 }
 ```
 
-When you define the inputs of a simulation, you should in general define the period for which you are setting values (except for variables which cannot change over time, such as a birth date).
+## Adding information to entities
```

Comment:
Not sure specifying the input variables is best described as "Adding information to entities". "Describing the simulation input" ?

Reply:
I don't know if "simulation input" is understandable.
I reworked the titles : 
Using the /calculate endpoint
- Describing the situation
-- Describing entities
-- Adding information to entities
- Computing a variable
- Understanding the result